src/Juvix/Compiler/Core/Transformation/Eta.hs

```diff
@@ -8,71 +8,65 @@ import Data.HashMap.Strict qualified as HashMap
 import Juvix.Compiler.Core.Extra
 import Juvix.Compiler.Core.Transformation.Base
 
+substDrop :: [Node] -> [Node] -> [Node]
```

Comment:
add documentation

Reply:
Done. I also moved this function to Extra.Utils.